### DISCUSSION TITLE
Wallet

### DISCUSSION THREAD

**Discussion by sikohima** [Chia-Network/chia-blockchain#4630] [Support]
How can i mine chia crypto and how can i get my money from wallet to external wallet like trust wallet

**Comment by ravi-18-Apple** [comment]
connect vpn to an exchange and then  transfer ERC20 from your trust wallet to any exchange or you can mine in china and store your crypto in trust wallet and you can transfer to any exchange then P2P your money is on your bank !! any issue msg me

**Comment by BrandtH22** [comment]
@sikohima , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
